src/test/javascript/spec/component/shared/metis/postings-reactions-bar/post-reactions-bar/post-reactions-bar.component.spec.ts

```diff
@@ -45,7 +48,7 @@ describe('PostReactionsBarComponent', () => {
                 { provide: ReactionService, useClass: MockReactionService },
                 { provide: AccountService, useClass: MockAccountService },
             ],
-            declarations: [PostReactionsBarComponent, MockPipe(ArtemisTranslatePipe)],
+            declarations: [PostReactionsBarComponent, MockPipe(ArtemisTranslatePipe), MockDirective(NgbTooltip), MockComponent(FaIconComponent)],
             schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
I don't know how to not add `schemas: [CUSTOM_ELEMENTS_SCHEMA]` when testing this component, as [template of the tested component](https://github.com/ls1intum/Artemis/pull/3907/files#diff-687f8b86b143c646398a2e817786ff23f505ab1db0b07d42d5b76d23a9c5569f) includes the web components `emoji-mart` and `ngx-emoji` which I cannot mock.

Reply:
fixed in this PR https://github.com/ls1intum/Artemis/pull/3928 ✅